index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Design Teardown</title>
+</head>
+
+<body>
+    <header class="main-header">
+        <div class="header-items">
+            <h2 class="logo">
+                Logo
+            </h2>
+            <ul>
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+            </ul>
```

Comment:
Kindly wrap the elements with ```<div>```
![kr1](https://user-images.githubusercontent.com/46388113/74500504-fba7b600-4ede-11ea-888e-aab3d5003da5.PNG)